app/scripts/metamask-controller.js

```diff
@@ -1549,6 +1553,11 @@ export default class MetamaskController extends EventEmitter {
     return keyring;
   }
 
+  async attemptLedgerTransportCreation() {
+    const keyring = this.getKeyringForDevice('ledger');
```

Comment:
This needs an `await`:

```
const keyring = await this.getKeyringForDevice('ledger');
```

Reply:
this is addressed since my latest rebase